libs/langchain/langchain/document_loaders/readthedocs.py

```diff
@@ -15,6 +15,7 @@ def __init__(
         errors: Optional[str] = None,
         custom_html_tag: Optional[Tuple[str, dict]] = None,
         patterns: Sequence[str] = ("*.htm", "*.html"),
+        exclude_index_pages: bool = False,
```

Comment:
what if this was `exclude_links_ratio: Optional[float] = None`

None -> equivalent to exclude_index_pages = False
float -> equivalent to exclude_index_pages = True and you can control what the ratio threshold for exclusion is

Reply:
Great idea! Changing.